packages/server/routes/tables.js

```diff
@@ -589,16 +589,20 @@ router.get(
           ),
       };
     }
+    // Table Data card
     const dataCard = div(
       { class: "d-flex text-center" },
       div({ class: "mx-auto" }, h4(`${nrows}`), req.__("Rows")),
       div(
         { class: "mx-auto" },
-        a(
-          { href: `/list/${table.name}` },
-          i({ class: "fas fa-2x fa-edit" }),
+        a( // TBD Decide about edit of users table data - currently doesnt work - I had put link to useradmin
```

Comment:
Why doesn't edit users work - works fine for me on the current master branch. What problems do you see?

Reply:
> Why doesn't edit users work - works fine for me on the current master branch. What problems do you see?
Please see video with the problem https://recordit.co/BeCbHLJ5Ps

Reply:
ok i see. let's go with your solution (redirect) until this can be fixed